modules/ppcp-button/resources/js/button.js

```diff
@@ -121,10 +121,22 @@ const bootstrap = () => {
             return actions.reject();
         }
     };
-    const onSmartButtonsInit = () => {
+
+    let smartButtonsOptions = {
+        onInit: null,
```

Comment:
Maybe it would be better to make it an array. But probably not important for now.

Reply:
I also thought on that, but didn't want to over engineer it prematurely. Probably this object should also have its own separate file/class.